Dockerfile.bundle

```diff
@@ -1,8 +1,8 @@
-FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.23 as builder
+FROM registry.redhat.io/ubi9/ubi-minimal@sha256:e1c4703364c5cb58f5462575dc90345bcd934ddc45e6c32f9c162f2b5617681c as builder
```

Comment:
let me double check, but IIRC moving to rhel9 as a base was a necessary prerequisite for FIPS